PhysicsTools/PatAlgos/python/slimming/packedPFCandidates_cfi.py

```diff
@@ -29,6 +31,19 @@
 from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
 run2_miniAOD_80XLegacy.toModify(packedPFCandidates, chargedHadronIsolation = "" )
 
+from Configuration.Eras.Modifier_run2_HCAL_2018_cff import run2_HCAL_2018
+run2_HCAL_2018.toModify(packedPFCandidates,
+    pfCandidateTypesForHcalDepth = cms.vint32(130,11,22,211,13),  # PF cand types for adding Hcal depth energy frac information
+                        # (130: neutral h, 11: ele, 22: photon, 211: charged h, 13: mu) # excluding e.g. 1:h_HF, 2:egamma_HF
+    storeHcalDepthEndcapOnly = cms.bool(True)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

```python
pfCandidateTypesForHcalDepth = [130,11,22,211,13], 
storeHcalDepthEndcapOnly = True
```